Scripts/Editor/AssetBundleWindow.cs

```diff
@@ -112,6 +119,107 @@ void OnGUI()
                 {
                     Directory.CreateDirectory(Application.dataPath + exportLocation);
                 }
+
+                //clear output folder
+                System.IO.DirectoryInfo di = new DirectoryInfo(Application.dataPath + exportLocation);
```

Comment:
Clears the output folder to avoid folder/file already exists errors, not sure if these really arise, but I figured I'd cover my bases.